src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,92 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Browsing Installed Applications
+
+Installed applications can be viewed in the Applications menu and the Pop!\_Shop Installed screen.
+
+### The Applications Menu
+
+The Applications menu can be accessed from the Dock or by clicking `Applications` in the upper left corner of the screen.
+
+![Launch Applications Menu](/images/locate-launch-applications/launch-applications-menus.png)
+
+Applications can be grouped into folders. Click `Create Folder` and give the folder a name.
+
+![Create Application Folder](/images/locate-launch-applications/create-application-folder.png)
+
+Click and drag an application icon to add it to your folder.
+
+<video autoplay loop>
+    <source src="/images/locate-launch-applications/drag-application.webm" />
+</video>
+
+### The Pop!\_Shop
+
+Launch the Pop!\_Shop, then click the `Installed` button.
+
+![Pop Shop Installed Apps](/images/locate-launch-applications/pop-shop-installed-apps.png)
+
+## Launching Applications
+
+Applications can be launched from the Dock, using the Launcher, the Applications menu, and from the Terminal.
+
+### Using the Launcher
+
+Press the `SUPER` key or the Show Launcher icon to bring up the Launcher.
+
+![Show Launcher](/images/locate-launch-applications/show-launcher.png)
+
+Search for the name of the application, then hit `Enter`.
```

Comment:
You don't think it would be helpful to explain somehow that existing windows will show up above running applications, and that the arrow keys can be used to launch an app instead of focusing a window? Not a hard requirement, just seems like the kind of detail that a manual could elaborate on vs. having the user learn it through trial-and-error. (Given how much the Launcher does and how much of its functionality is visually hidden behind keywords/prefixes, an entire page on just the Launcher might be more appropriate, and if you go that route, it could always be linked from here as a "more info" kind of thing.)

Reply:
There will be dedicated sections describing these components (COSMIC, the Launcher, kernelstub, etc.) in the Introduction section. Let me know what additional components should go there.

Reply:
And I'll address the top result feature

Reply:
> There will be dedicated sections describing these components (COSMIC, the Launcher, kernelstub, etc.)

Kernelstub isn't really related to the GUI at all like the other two components you mentioned are. Additionally, an extensive reference for all of the Launcher's functionality would be more than an introduction. I guess we'll have to see how that works.

Reply:
Just to give an idea of what's included in the Launcher:
- Switching between open windows
- Launching applications
- Browsing the filesystem
- Searching the home directory
- Searching recently-opened files
- Performing mathematic calculations
- Performing web searches
- Running terminal commands in the background & foreground
- Adjusting volume, rebooting/logging out/etc, running arbitrary scripts

And many of these have special prefixes you need to know (e.g. each web search provider has a prefix, similar to a keyword, that you need to type to select which provider to use.)

Reply:
Good to know, are these procedures documented anywhere? This is what I could find in the Pop repo: https://github.com/pop-os/launcher